lib/client/src/client.rs

```diff
@@ -103,6 +107,7 @@ impl Client {
         commitment: CommitmentConfig,
         fee_payer: Arc<Keypair>,
         timeout: Option<Duration>,
+        fallback_oracle_config: Option<FallbackOracleConfig>,
```

Comment:
please don't change ::new() (unnecessary api change for users; and ::new() is deprecated over using a builder anyway)